components/suggest/src/store.rs

```diff
@@ -1990,6 +2032,24 @@ mod tests {
                 },
                 expect![[r#"
                     [
+                        Pocket {
```

Comment:
I think this test is getting a bit too long...and I think we might also be missing some test coverage that would've caught the missing `DESC` in the first query. We can save this for a follow-up, though, unless you'd like to tackle it now!

To make sure we're covering all the cases:

Let's add one test, like `amp_wikipedia_scores`, that ingests AMP and Wikipedia suggestions. This test can exercise sorting and limits for the query that just selects from `keywords`. We can have it ingest two AMP suggestions—one with a score of 0.3, and one with 0.1—and a Wikipedia suggestion without a score.

We can then verify that the suggestions are sorted correctly, with and without a limit: AMP with 0.3, Wikipedia with the default of 0.2, and AMP with 0.1. I know we won't see AMP suggestions like that IRL (yet?); this is more to verify that the `ORDER BY` works, and that `ORDER BY` combined with a `LIMIT` returns them in the right order.

Let's have our second test, `wikipedia_pocket_scores`, ingest Wikipedia and Pocket suggestions. This test can exercise sorting and limits for the query that selects from `prefix_keywords` and `keywords`.

We can follow the same "sandwich" approach for this second test as for the first: a Pocket suggestion with a score higher than 0.2, a Wikipedia suggestion without a score, and a second Pocket suggestion with a lower score.